README.md

```diff
@@ -207,6 +207,12 @@ to get the hang of things.
 
 ## Configuration
```

Comment:
This last commit is not just a docs update. If there are changes to tidy other commits, they belong combined with those commits. That will ensure that all the linting and tests passes on each commit individually, rather than just over the branch.

If you have other changes, they belong in new commits specific to those improvements.